src/mame/drivers/midvunit.cpp

```diff
@@ -1287,6 +1309,14 @@ ROM_START( crusnusa40 ) /* Version 4.0, Wed Feb 08 1995 - 10:45:14 */
 	ROM_LOAD32_BYTE( "cusa.u27",     0x800001, 0x80000, CRC(2d977a8e) SHA1(8f4d511bfd6c3bee18daa7253be1a27d079aec8f) )
 	ROM_LOAD32_BYTE( "cusa.u28",     0x800002, 0x80000, CRC(cffa5fb1) SHA1(fb73bc8f65b604c374f88d0ecf06c50ef52f0547) )
 	ROM_LOAD32_BYTE( "cusa.u29",     0x800003, 0x80000, CRC(cbe52c60) SHA1(3f309ce8ef1784c830f4160cfe76dc3a0b438cac) )
+	
+/*	ROM_REGION( 0x0b33, "pals", 0 ) // all protected
```

Comment:
Any particular reason why you're adding all the pals commented out?

Reply:
I would like to say documentation but it could be more than that. Only crusnusa had the pal dumps which are currently marked as bad. Commenting them out didn't make a difference in gameplay. Alternatively, I could uncomment them and mark all pals as "no dump" which was the case in an earlier commit (since the current dumps are useless to me anyway). I initially thought they were related to networking game pcbs but now that I know how these pcbs network each other, it's most likely possible without the pals.

Reply:
If you know those pals are present on the real PCBs and aren't dumped, then adding them as NO_DUMP is the correct way to go, but commenting them out doesn't make any sense to me. I'd ask you to uncomment them.

Reply:
Done. I should note one pal, A-19668, has its name shared across all three games but I have a feeling the contents may differ across each game.

Reply:
That'd be the DSP control PAL.

Reply:
@Risugami It would be nice to document what these pals do to implement temporary simulation for them until pal emulation is understood (although probably not so much needed to link this hardware). Again I ask for how you took a stab at communications for Midway V Unit (we need to chat more often).

Reply:
https://imgur.com/a/drBoljV someone more experienced with MAME's... everything... would have to look over doing this. The entire machine had to be butchered to even allow this much and they are still bursting data at each other without giving the other a chance at all. This was at 1/4th CPU speed as at full speed watchdog kills it.

Reply:
@Risugami The main processor most likely has features that aren't quite understood. After all, the main processor is responsible for communications. If only I could see the code you implemented.